unic/ucd/hangul/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "unic-ucd-hangul"
+version = "0.7.0"
+authors = ["The UNIC Project Developers"]
+repository = "https://github.com/behnam/rust-unic/"
+license = "MIT/Apache-2.0"
+description = "UNIC — Unicode Character Database — Hangul"
```

Comment:
Could you please elaborate on what the create is here?

Maybe "Hangul Syllable Composition & Decomposition", to follow the name of the algorithms in TUS (The Unicode Standard)?

Reply:
Fixed :)